### PR TITLE
Add mx-puppet-groupme support

### PR DESCRIPTION
**Pull request by xangelix**
Adds support for: https://gitlab.com/robintown/mx-puppet-groupme

Modeled after the other mx-puppet bridges.

WIP as additional testing needs to be done--however, there are no currently known issues.

### CODE REVIEW

File: group_vars/matrix_servers [hunk 1]
```diff
@@ -622,6 +622,41 @@ matrix_mx_puppet_steam_database_password: "{{ matrix_synapse_macaroon_secret_key
...
+    ['docker.service']
+    +
+    (['matrix-synapse.service'] if matrix_synapse_enabled else [])
+    +
+    (['matrix-postgres.service'] if matrix_postgres_enabled else [])
+  }}
+
+matrix_mx_puppet_groupme_appservice_token: "{{ matrix_synapse_macaroon_secret_key | password_hash('sha512', 'mxste.as.tok') | to_uuid }}"
+
+matrix_mx_puppet_groupme_homeserver_token: "{{ matrix_synapse_macaroon_secret_key | password_hash('sha512', 'mxste.hs.tok') | to_uuid }}"
```
Comment (spantaleev) [inline group_vars/matrix_servers:647]: These should be changed to something unique. Right now it conflicts with the Steam bridge.

Perhaps `mxgro.as.tok` and `mxgro.hs.tok`.

Comment (xangelix) [reply group_vars/matrix_servers]: Missed that one, thanks.

**Comment by xangelix** [comment]
Docker build is working well and everything else seems to be in order.

**Comment by spantaleev** [comment]
Looks great! Thanks! :+1:

**Pull request merged**